static/app/components/events/contextSummary/contextIcon.tsx

```diff
@@ -61,7 +61,7 @@ const LOGO_MAPPING = {
   'net-core': logoNetcore,
   'net-framework': logoNetframework,
   'qq-browser': logoQq,
-  nintendo: logoNintendo,
+  'nintendo-os': logoNintendo,
```

Comment:
The way this works is that if sdk passes `Nintendo OS` we convert that to `nintendo-os` and then look for an image that is named that.